server/app/services/export/PdfExporter.java

```diff
@@ -104,6 +118,10 @@ private byte[] buildPDF(
               "Status: " + statusValue.orElse("none"),
               FontFactory.getFont(FontFactory.HELVETICA_BOLD, 12));
       document.add(status);
+      Paragraph submitTimeInformation =
+          new Paragraph(
+              "Submit Time: " + submitTime, FontFactory.getFont(FontFactory.HELVETICA_BOLD, 12));
+      document.add(submitTimeInformation);
```

Comment:
```suggestion
if (submitTime.isPresent()) {
      Paragraph submitTimeInformation =
          new Paragraph(
              "Submit Time: " + submitTime, FontFactory.getFont(FontFactory.HELVETICA_BOLD, 12));
      document.add(submitTimeInformation);
}      
```

Reply:
I think Submit Time should be there like status- Today even if status is empty, we say Status.orElse("none")

Reply:
In that case go with something like "Not yet submitted" instead of "Submit Time: date-time-here". It'll be more readable.

Reply:
sure thing